modal/cli/run.py

```diff
@@ -77,10 +77,10 @@ class CliRunnableSignature:
 
 def safe_get_type_hints(func_or_cls: typing.Union[Callable[..., Any], type]) -> dict[str, type]:
     try:
-        return get_type_hints(func_or_cls)
+        return typing.get_type_hints(func_or_cls)
     except Exception as exc:
         # E.g., if entrypoint type hints cannot be evaluated by local Python runtime
-        msg = "Unable to generate command line interface for app entrypoint. See traceback above for details."
+        msg = "Unable to generate command line interface for app entrypoint due to unparseable type hints:\n" + str(exc)
```

Comment:
This is tough to communicate about in general, but "unparseable" might be misleading in some contexts right? There are type annotations that can be "parsed" in some narrow sense but that we can't turn into a CLI option.